external-docs/docs/guides/command-line/inspect-command.md

```diff
@@ -1,3 +1,939 @@
+* Data cube with multiple units and multiple measures
+* Code list
+  
+## Output format
+
+In the following, the format of the outputs generated by the inspect command is discussed.
+
+### Data cube
+
+`csvcubed inspect --log-level info ./datacube.csv-metadata.json`
+
+```
+Successfully parsed csvw json-ld to rdf graph.
+Successfully loaded 'x' table schemas into the rdf graph.
+Successfully loaded csv into dataframe.
+
+- This file is a data cube.
+
+- The data cube has the following catalog metadata:
+        - Title: 
+        - Label:
+        - Issued:
+        - Modified: 
+        - License: 
+        - Creator:
+        - Publisher:
+        - Landing Pages:
+        - Themes:
+        - Keywords:
+        - Contact Point:
+        - Identifier:
+        - Comment:
+        - Description:
+
+- The data cube has the following data structure definition:
+        - Dataset Label:
+        - Number of Components:
+        - Components:
+        - Columns where suppress output is true:
+
+- The data cube has the following code list information:
+        - Number of Code Lists:
+        - Code Lists:
+
+- The data cube has the following dataset information:
+        - Number of Observations:
+        - Number of Duplicates:
+        - First 10 Observations:
+        - Last 10 Observations:
+        
+- The data cube has the following value counts:
+        - Value counts broken-down by measure and unit (of measure):
+```
+
+* `This file is a data cube` - whether the input CSV-W is of type data cube or code list.
+  
+* `The data cube has the following catalog metadata`:
+
+  | Field         | Description                                                         |
+  | ------------- | ------------------------------------------------------------------- |
+  | Title         | Title of the data set.                                              |
+  | Label         | Label of the data set.                                              |
+  | Issued        | Data set issued timestamp in the `YYYY-DD-MM'T'HH:mm:ssZ` format.   |
+  | Modified      | Data set modified timestamp in the `YYYY-DD-MM'T'HH:mm:ssZ` format. |
+  | License       | License information.                                                |
+  | Creator       | Creator information web page url.                                   |
+  | Publisher     | Publisher information web page url.                                 |
+  | Landing Pages | List of web urls of landing pages.                                  |
+  | Themes        | List of web urls of theme information.                              |
+  | Keywords      | List of keywords.                                                   |
+  | Contact Point | Contact information.                                                |
+  | Identifier    | Data set identifier.                                                |
+  | Comment       | Any comments associated with the data set.                          |
```

Comment:
Can we re-name `Comment` to `Summary`? It'd make it more consistent with the way they provide the information in the first place.